components/ide/jetbrains/gateway-plugin/src/main/kotlin/io/gitpod/jetbrains/gateway/GitpodSettingsConfigurable.kt

```diff
@@ -31,6 +31,11 @@ class GitpodSettingsConfigurable : BoundConfigurable("Gitpod") {
                     .comment("Helpful if you are behind a firewall/proxy that blocks SSH or " +
                             "have complicated SSH setup (bastions, proxy jumps, etc.)")
             }
+            row {
+                checkBox("Persistent connection heartbeats")
+                    .bindSelected(state::additionalHeartbeat)
+                    .comment("Keep workspaces running as long as the IDE connection remains active.")
```

Comment:
nit: we don't end with a period in the checkbox above, and neither do we in the Dashboard. 

```suggestion
                    .comment("Keep workspaces running as long as the IDE connection remains active")
```